spec/features/passenger_management_spec.rb

```diff
@@ -1,12 +1,29 @@
 require 'rails_helper' 
 
 feature 'Passenger Management' do 
+  before :each do
+    sign_in(create :user, :admin)
+  end
   scenario 'creating a new passenger' do 
+    visit passengers_path
+    click_link 'Add New Passenger'
+    fill_in('passenger[name]', with: 'Foo Bar')
```

Comment:
Normally for these if the form is constructed correctly you can say `fill_in 'Name', with: 'Foo Bar'`.